examples/sod-mpi.py

```diff
@@ -152,9 +183,22 @@ def my_checkpoint(step, t, dt, state):
     if current_t - t_final < 0:
         raise ValueError("Simulation exited abnormally")
 
+    if use_logmgr:
+        logmgr.close()
+    elif use_profiling:
+        print(actx.tabulate_profiling_data())
+
 
 if __name__ == "__main__":
     logging.basicConfig(format="%(message)s", level=logging.INFO)
-    main()
+    import argparse
+    parser = argparse.ArgumentParser(description="Sod (MPI version)")
+    parser.add_argument("--profile", action="store_true",
+        help="enable kernel profiling")
+    parser.add_argument("--logging", action="store_true",
+        help="enable time series logging")
+    args = parser.parse_args()
+
+    main(use_profiling=args.profile, use_logmgr=args.logging)
```

Comment:
Not a fan. The driver should act as the "configuration file". Not sure I like having more "command-line configurability".

Reply:
Replaced with two config bools instead of argument parsing.